pkg/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	semver "github.com/blang/semver/v4"
+)
+
+const (
+	ReasonIncompatibleOperatorsInstalled = "IncompatibleOperatorsInstalled"
+	MaxOpenShiftVersionProperty          = "olm.maxOpenShiftVersion"
+	OwnerKindKey                         = "olm.operatorframework.io/owner-kind"
+	OwnerNameKey                         = "olm.operatorframework.io/owner-name"
+	PackageNameKey                       = "olm.operatorframework.io/package-name"
+	BundleNameKey                        = "olm.operatorframework.io/bundle-name"
+	BundleVersionKey                     = "olm.operatorframework.io/bundle-version"
```

Comment:
I'd move these back to the incompatibleOperatorController file and unexport them all.